presto-verifier/src/main/java/com/facebook/presto/verifier/source/VerifierDao.java

```diff
@@ -35,12 +35,13 @@
             "  control_query text NOT NULL,\n" +
             "  control_username varchar(256) DEFAULT NULL,\n" +
             "  control_password varchar(256) DEFAULT NULL,\n" +
+            "  control_session_properties text DEFAULT NULL,\n" +
             "  test_catalog varchar(256) NOT NULL,\n" +
             "  test_schema varchar(256) NOT NULL,\n" +
             "  test_query text NOT NULL,\n" +
             "  test_username varchar(256) DEFAULT NULL,\n" +
             "  test_password varchar(256) DEFAULT NULL,\n" +
-            "  session_properties_json text DEFAULT NULL)")
+            "  test_session_properties text DEFAULT NULL)")
```

Comment:
@caithagoras This change needs to be reflected in the docs, right? https://prestodb.io/docs/current/installation/verifier.html